docs/en/observability/synthetics-settings.asciidoc

```diff
@@ -10,11 +10,14 @@ There are several Synthetics settings you can adjust in {kib}.
 Alerting enables you to detect complex conditions using *rules* across Observability apps
 and send a notification using *connectors*.
 
-When you create a new synthetic monitor, a new default synthetics rule will be applied.
-To edit the default rule:
+When you create a new synthetic monitor, new default synthetics rules will be applied.
+To edit the default rules:
 
 . Click *Alerts and rules* in the top bar.
-. Click *Monitor status rule* to open a panel where you can edit the rule's configuration.
+. Select a rule to open a panel where you can edit the rule's configuration:
+** *Monitor status rule* for receiving notifications for errors and outages.
+** *TLS certificate rule* for receiving notifications when one or more of your monitors has a
```

Comment:
Might be worth calling out here it's only for certain lightweight checks, so users don't think it includes browser monitors domain certs, for example:
> ...for receiving notifications when one or more of your HTTP or TCP lightweight monitors has a...